openverse_catalog/dags/providers/provider_api_scripts/jamendo.py

```diff
@@ -94,7 +94,8 @@ def _get_items():
     while should_continue:
         query_params = _get_query_params(offset=offset)
         batch_data = _get_batch_json(query_params=query_params)
-        if isinstance(batch_data, list) and len(batch_data) > 0:
+        logger.info(f"batch_data: {batch_data}, query_params: {query_params}")
```

Comment:
`batch_data` is going to be a pretty large list in most cases, do you think this could be a `logger.debug` line?

Reply:
It was probably left their by mistake :)